content/projects/oop/bank-accounts/part-1/_index.md

```diff
@@ -133,4 +130,8 @@ Also, please make use of assertions within your code to make sure that errors ar
 - withdraw a negative amount
 - deposit a negative amount
 - withdraw more money than you have
-- set a negative interest rate
\ No newline at end of file
+- set a negative interest rate
+
+**NB: Javascript assertions**
+
+Please make use of [nodejs assert()](https://www.w3schools.com/nodejs/met_assert.asp) because [console.assert()](<https://developer.mozilla.org/en-US/docs/Web/API/console/assert#:~:text=The%20console.assert()%20method%20writes%20an%20error%20message%20to%20the%20console%20if%20the%20assertion%20is%20false.%20If%20the%20assertion%20is%20true%2C%20nothing%20happens.>) does not throw errors.
```

Comment:
Can you please add a bit more of an explanation here? We want to discourage cargo-culting so it is useful to tell people why a decision has been made so that they understand. Do you think the learners will understand the reason behind this? Or do you think they will just do what they are told without understanding?